### ISSUE TITLE
 Error creating machine: Error in driver during machine creation

### ISSUE THREAD

**Issue by BranOfStark** [code-ready/crc#1745]
### General information

  * OS: RHEL 7.9
  * Hypervisor: KVM 
  * Did you run `crc setup` before starting it (Yes/No)? Yes

## CRC version
```bash
CodeReady Containers version: 1.19.0+94b0362
OpenShift version: 4.6.3 (embedded in executable)
```
  
## CRC status
```bash
[cloud-user@rhel79 crc-install]$ ./crc status
CRC VM:          Stopped
OpenShift:       Stopped
Disk Usage:      0B of 0B (Inside the CRC VM)
Cache Usage:     15.32GB
Cache Directory: /home/cloud-user/.crc/cache
```

## CRC config
```bash
No output
```

## Host Operating System
```bash
[cloud-user@rhel79 crc-install]$ cat /etc/os-release
NAME="Red Hat Enterprise Linux Server"
VERSION="7.9 (Maipo)"
ID="rhel"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="7.9"
PRETTY_NAME="Red Hat Enterprise Linux"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:7.9:GA:server"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 7"
REDHAT_BUGZILLA_PRODUCT_VERSION=7.9
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="7.9"
```

### Steps to reproduce

1. Trying to setup and run openshift cluster on a RHEL 7.9 instance on OCI cloud.
2. crc setup step executes successfully.  
3. On running the crc start command, received the following error 
Error creating machine: Error creating the VM: Error creating machine: Error in driver during machine creation: virError(Code=1, Domain=10, Message='internal error: process exited while connecting to monitor: qemu-kvm: -machine q35,accel=kvm,usb=off,dump-guest-core=off: Unsupported machine type
Use -machine help to list supported machines!')**
 

### Expected
To start the VM for OC

### Actual
Error creating machine: Error creating the VM: Error creating machine: Error in driver during machine creation: virError(Code=1, Domain=10, Message='internal error: process exited while connecting to monitor: qemu-kvm: -machine q35,accel=kvm,usb=off,dump-guest-core=off: Unsupported machine type

### Logs
DEBU CodeReady Containers version: 1.19.0+94b0362
DEBU OpenShift version: 4.6.3 (embedded in executable)
DEBU Total memory of system is 31453016064 bytes
DEBU No new version available. The latest version is 1.19.0
DEBU Couldn't set key CPE_NAME, no corresponding struct field found
DEBU Couldn't set key , no corresponding struct field found
DEBU Couldn't set key REDHAT_BUGZILLA_PRODUCT, no corresponding struct field found
DEBU Couldn't set key REDHAT_BUGZILLA_PRODUCT_VERSION, no corresponding struct field found
DEBU Couldn't set key REDHAT_SUPPORT_PRODUCT, no corresponding struct field found
DEBU Couldn't set key REDHAT_SUPPORT_PRODUCT_VERSION, no corresponding struct field found
INFO Checking if podman remote executable is cached
DEBU Currently podman remote is not supported
INFO Checking if goodhosts executable is cached
DEBU goodhost executable already cached
INFO Checking minimum RAM requirements
DEBU Total memory of system is 31453016064 bytes
INFO Checking if running as non-root
INFO Checking if Virtualization is enabled
DEBU Checking if the vmx/svm flags are present in /proc/cpuinfo
DEBU CPU virtualization flags are good
INFO Checking if KVM is enabled
DEBU Checking if /dev/kvm exists
DEBU /dev/kvm was found
INFO Checking if libvirt is installed
DEBU Checking if 'virsh' is available
DEBU 'virsh' was found in /usr/bin/virsh
DEBU Checking 'virsh capabilities' for libvirtd/qemu availability
DEBU Running 'virsh capabilities'
DEBU Found x86_64 hypervisor with 'hvm' capabilities
INFO Checking if user is part of libvirt group
DEBU Checking if current user is part of the libvirt group
DEBU Running '/usr/bin/groups cloud-user'
DEBU Current user is already in the libvirt group
INFO Checking if libvirt daemon is running
DEBU Checking if libvirtd service is running
DEBU Running 'systemctl status virtqemud.socket'
DEBU Command failed: exit status 4
DEBU stdout:
DEBU stderr: Unit virtqemud.socket could not be found.
DEBU virtqemud.socket is not running
DEBU Running 'systemctl status libvirtd.socket'
DEBU Command failed: exit status 4
DEBU stdout:
DEBU stderr: Unit libvirtd.socket could not be found.
DEBU libvirtd.socket is not running
DEBU Running 'systemctl status virtqemud.service'
DEBU Command failed: exit status 4
DEBU stdout:
DEBU stderr: Unit virtqemud.service could not be found.
DEBU virtqemud.service is not running
DEBU Running 'systemctl status libvirtd.service'
DEBU libvirtd.service is running
INFO Checking if a supported libvirt version is installed
DEBU Checking if libvirt version is >=3.4.0
DEBU Running 'virsh -v'
INFO Checking if crc-driver-libvirt is installed
DEBU Checking if crc-driver-libvirt is installed
DEBU Running '/home/cloud-user/.crc/bin/crc-driver-libvirt version'
DEBU Found crc-driver-libvirt version 0.12.12
DEBU crc-driver-libvirt is already installed
INFO Checking if libvirt 'crc' network is available
DEBU Checking if libvirt 'crc' network exists
DEBU Running 'virsh --connect qemu:///system net-info crc'
DEBU Checking if libvirt 'crc' definition is up to date
DEBU Running 'virsh --connect qemu:///system net-dumpxml --inactive crc'
DEBU libvirt 'crc' network has the expected value
INFO Checking if libvirt 'crc' network is active
DEBU Checking if libvirt 'crc' network is active
DEBU Running 'virsh --connect qemu:///system net-info crc'
DEBU libvirt 'crc' network is already active
INFO Checking if NetworkManager is installed
DEBU Checking if 'nmcli' is available
DEBU 'nmcli' was found in /usr/bin/nmcli
INFO Checking if NetworkManager service is running
DEBU Checking if NetworkManager.service is running
DEBU Running 'systemctl status NetworkManager'
DEBU NetworkManager.service is already running
INFO Checking if /etc/NetworkManager/conf.d/crc-nm-dnsmasq.conf exists
DEBU Checking NetworkManager configuration
DEBU NetworkManager configuration is good
INFO Checking if /etc/NetworkManager/dnsmasq.d/crc.conf exists
DEBU Checking dnsmasq configuration
DEBU dnsmasq configuration is good
Checking file: /home/cloud-user/.crc/machines/crc/.crc-exist
Found binary path at /home/cloud-user/.crc/bin/crc-driver-libvirt
Launching plugin server for driver libvirt
Plugin server listening at address 127.0.0.1:38656
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
(crc) Calling .GetBundleName
(crc) Calling .GetState
(crc) DBG | Getting current state...
(crc) DBG | Fetching VM...
INFO Starting CodeReady Containers VM for OpenShift 4.6.3...
DEBU Updating CRC VM configuration
(crc) Calling .GetConfigRaw
(crc) Calling .Start
(crc) DBG | Starting VM crc
(crc) DBG | Validating network
(crc) DBG | Validating storage pool
(crc) Failed to start: virError(Code=1, Domain=10, Message='internal error: process exited while connecting to monitor: qemu-kvm: -machine q35,accel=kvm,usb=off,dump-guest-core=off: Unsupported machine type
(crc) Use -machine help to list supported machines!')
Making call to close driver server
(crc) Calling .Close
Successfully made call to close driver server
Making call to close connection to plugin binary
(crc) DBG | Closing plugin on server side
Error starting stopped VM: virError(Code=1, Domain=10, Message='internal error: process exited while connecting to monitor: qemu-kvm: -machine q35,accel=kvm,usb=off,dump-guest-core=off: Unsupported machine type
Use -machine help to list supported machines!')

You can start crc with `crc start --log-level debug` to collect logs.
Please consider posting this on http://gist.github.com/ and post the link in the issue.

**Comment by linuxqiao** [comment]
Same problem on 7.9 GA kernel, anyone can help?

**Comment by cfergeau** [comment]
This is a duplicate of #1708
No great solution for 1.19, the various options are to stay with 1.18, wait for 1.20 (coming next week), upgrade to rhel8, ...
